docs/whats_new.rst

```diff
@@ -27,6 +27,7 @@ Version 0.5.0 (Unreleased)
 Changelog
 ~~~~~~~~~
 - :meth:`~pyprep.NoisyChannels.find_bad_by_nan_flat` now accepts a ``flat_threshold`` argument, by `Nabil Alibou`_ (:gh:`144`)
+- changed _mad function in utils.py to use median_abs_deviation from the sciPy module, by `Ayush Agarwal`_ (:gh:`153`).
```

Comment:
looks good, except that you need to add a link to your name at the bottom of this file, see: `.. _Stefan Appelhoff: https://stefanappelhoff.com/` in the file --> just adjust this to fit for your name!